pipeline/mist.go

```diff
@@ -64,29 +68,22 @@ func (m *mist) HandleStartUploadJob(job *JobInfo) (*HandlerOutput, error) {
 	log.AddContext(job.RequestID, "mist_target_url", mistTargetURL)
 	log.AddContext(job.RequestID, "segmented_url", job.SegmentingTargetURL)
 
-	// Arweave / IPFS URLs don't support HTTP Range requests and so Mist can't natively handle them for segmenting
-	// This workaround copies the file from Arweave to S3 and then tells Mist to use the S3 URL
-	if clients.IsDStorageResource(job.SourceFile) {
-		if !isVideo(job.RequestID, job.SourceFile) {
-			return nil, fmt.Errorf("source was not a video: %s", job.SourceFile)
-		}
-		sourceURL, err := url.Parse(job.SourceFile)
-		if err != nil {
-			return nil, fmt.Errorf("failed to parse source as URL: %w", err)
-		}
+	sourceURL, err := url.Parse(job.SourceFile)
```

Comment:
For the mist case, is it able to handle a pre-signed URL from s3? I can't recall where we landed on that discussion.

Reply:
That's a good point, thanks for the reminder. I have tested and it is working for mist, but i need to double check I understand fully.

Reply:
Ok i've ensured that we now send mist the same type of URL we used to (rather than the presigned one). e.g. what we do already here: https://github.com/livepeer/catalyst-api/blob/2d89524e60a33e95453e2583d09e5e0efe37c513/pipeline/mist.go#L87